### PR TITLE
dev-java/pdf-renderer: update java

### PR DESCRIPTION
**Pull request by vaukai**
Package-Manager: Portage-3.0.17, Repoman-3.0.2
Signed-off-by: Volkmar W. Pogatzki <gentoo@pogatzki.net>

### CODE REVIEW

File: dev-java/pdf-renderer/pdf-renderer-1.0.5-r1.ebuild [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+HOMEPAGE="https://wiki.gentoo.org/wiki/No_homepage"
+SRC_URI="https://repo1.maven.org/maven2/org/swinglabs/pdf-renderer/1.0.5/pdf-renderer-1.0.5-sources.jar"
+
+LICENSE="LGPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc64 ~x86"
+
+RDEPEND=">=virtual/jre-1.8:*"
+DEPEND=">=virtual/jdk-1.8:*
+	app-arch/unzip"
```
Comment (fordfrog) [inline dev-java/pdf-renderer/pdf-renderer-1.0.5-r1.ebuild:20]: if `unzip` is only used to unpack the `jar` file then it should go to `BDEPEND`.

Comment (vaukai) [reply dev-java/pdf-renderer/pdf-renderer-1.0.5-r1.ebuild]: Done

**Comment by fordfrog** [comment]
@vaukai thanks! i did some more changes to your commit, see f1e644e146476a8830f464b64043f45048a27d1c. i also committed it straight to stable and remove the old version.

**Comment by vaukai** [comment]
> @vaukai thanks! i did some more changes to your commit, see f1e644e146476a8830f464b64043f45048a27d1c. i also committed it straight to stable and remove the old version.

Thank you

**Pull request closed without merging**